package.json

```diff
@@ -26,7 +26,7 @@
   "dependencies": {
     "core-js": "^2.5.6",
     "tslib": "^1.10.0",
-    "protractor": "^5.4.2",
+    "protractor": "^7.0.0",
     "karma-jasmine": "^2.0.1"
   },
   "devDependencies": {
```

Comment:
Consider to update the protractor dependencies such as @types/jasmine
The dependencies are published at https://www.npmjs.com/package/protractor > Dependencies